recipes/duckdb/all/patches/0.8.1-0001-fix-cmake.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
```

Comment:
It seems this patch is the same as the one for 0.8.0? I fail to see any difference

Reply:
@memsharded 
line numbers in tools/sqlite3_api_wrapper/CMakeLists.txt are different.

in 0.8.0
```
--- a/tools/sqlite3_api_wrapper/CMakeLists.txt
+++ b/tools/sqlite3_api_wrapper/CMakeLists.txt
@@ -19,7 +19,7 @@ if(NOT AMALGAMATION_BUILD)
 endif()
 link_threads(sqlite3_api_wrapper_static)
```

in 0.8.1
```
--- a/tools/sqlite3_api_wrapper/CMakeLists.txt
+++ b/tools/sqlite3_api_wrapper/CMakeLists.txt
@@ -26,7 +26,7 @@ if(NOT AMALGAMATION_BUILD)
 endif()
 link_threads(sqlite3_api_wrapper_static)
```

Reply:
Thanks for the clarification, I didn't realize this while reviewing!